### PR TITLE
Add way to remove project from recent projects list (addresses #966)

### PR DESCRIPTION
**Pull request by nirvn**
GIF:
Peek 2020-05-04 15-59

@m-kuhn , as discussed.

### CODE REVIEW

**Comment by nirvn** [comment]
@m-kuhn , @3nids , you might want to consider this for 1.6.

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-982-98c548c8e45423f95987f1043b1aa8d82116181a-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-982-98c548c8e45423f95987f1043b1aa8d82116181a-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-982-98c548c8e45423f95987f1043b1aa8d82116181a-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-982-98c548c8e45423f95987f1043b1aa8d82116181a-x86_64.apk)

File: src/core/qgismobileapp.cpp [hunk 1]
```diff
@@ -319,6 +319,20 @@ void QgisMobileapp::loadProjectQuirks()
     mLayerTreeCanvasBridge->setAutoSetupOnFirstLayer( true );
 }
 
+void QgisMobileapp::removeRecentProject( const QString &path )
+{
+  QList<QPair<QString, QString>> projects = recentProjects();
+  for ( int idx = 0; idx < projects.count() && idx < 5; idx++ )
```
Comment (3nids) [inline src/core/qgismobileapp.cpp:325]: why only < 5 ?

Comment (nirvn) [reply src/core/qgismobileapp.cpp:325]: Good point -- the max 5 isn't needed here, will update.

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-982-e26c41c48bfa57cf0d3ebe60d62760a89b0d1999-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-982-e26c41c48bfa57cf0d3ebe60d62760a89b0d1999-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-982-e26c41c48bfa57cf0d3ebe60d62760a89b0d1999-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-982-e26c41c48bfa57cf0d3ebe60d62760a89b0d1999-x86_64.apk)

**Comment by m-kuhn** [comment]
Looks like "addresses" does not "fix" and close. Was that on purpose?

**Comment by nirvn** [comment]
No, we can close as implemented/fixed.

**Pull request merged**